app/models/listing.rb

```diff
@@ -73,7 +73,9 @@ def evaluate_markdown
   end
 
   def modify_inputs
-    ActsAsTaggableOn.default_parser = ActsAsTaggableOn::TagParser
```

Comment:
Setting the default_parser caused conflict in testing so I have to change how Tag is parsed in Listing.

Reply:
> caused conflict in testing 

What do you mean by conflict?

Reply:
I should've expanded on that. When running listing (and related) specs and my new spec together, ie 

```
bundle exec rspec spec/lib/data_update_scripts/index_listings_to_elasticsearch_spec.rb spec/models/article_spec.rb:245
```

My new spec will fail. This is because if the code path went through Listing Model which will go through the code `ActsAsTaggableOn.default_parser = ActsAsTaggableOn::TagParser` , setting the default parser I'm trying to avoid, hence failing my spec (automatically removing non-alphanumerics).